packages/ui/vue/create-headless-vue/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@coveo/create-headless-vue",
+  "description": "Initialize a Vite+Vue+Coveo Headless project",
+  "authors": "Coveo",
+  "version": "0.0.1",
```

Comment:
When will this get bumped to v1 (if ever) ?

Reply:
Prior to the merge, I will manually release 0.0.1
Then, it will be 1.0.0 with @coveo/cli@2.0.0